src/translations.cpp

```diff
@@ -73,14 +73,19 @@ std::string locale_dir()
     return loc_dir;
 }
 
-void set_language()
+//if called without any parameters sets the language to the one selected in the options menu
+//if called with a parameter from code sets the language to that instead
+void set_language( std::string lang )
```

Comment:
Passing a string by copy is not really necessary as you are not changing this variable in the function body.